react/bingo/src/bingo/GameLogic.js

```diff
@@ -0,0 +1,221 @@
+import React from 'react';
+import './BingoApp.css';
+import { cleanup } from '@testing-library/react';
+
+class GameLogic extends React.Component {
+    constructor(props) {
+        super(props);
+        this.maskedChars = [];
+
+        this.state = {
+            attemptLetter: '',
+            failedLetters: '',
+            successLetters: '',
+            challengeWord: [],
+            maskedWord: [],
+            isGameOver : false,
+            bingoColor: [
+                'attempt-left',
+                'attempt-left',
+                'attempt-left',
+                'attempt-left',
+                'attempt-left'
+            ]
+        };
+
+    }
+    getBlankText = () => {
+        let challengeText = this.props.challenge;
+        let size = challengeText.length;
+        let challengeChars = challengeText.split('');
+        let isInit = true;
+        let maskedChars = [];
```

Comment:
use "const" for any variable holding a value which will/should not be changed.

Reply:
addressing this comment in the new project. Hence resolving the comment.